fix the queries for `Label Filters` and `Metrics Browser` for metrics with special characters (#141)
VictoriaMetrics/grafana-datasource: CHANGELOG.md

## tip

> * BUGFIX: correct the queries for `Label Filters` and `Metrics Browser` for metrics with special characters. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/140)

Reviewer: ```suggestion
* BUGFIX: allow the display of metrics with special characters in `Label Filters` and `Metrics Browser`. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/140)
```

Author: I'm not sure if this description is accurate. The issue was that when attempting to retrieve `label=values` for metrics with special characters, the characters were not properly escaped, resulting in the query returning an error.
